src/openrct2/common.h

```diff
@@ -135,10 +135,18 @@ using money64 = fixed64_1dp;
 #define FIXED_1DP(whole, fraction) FIXED_XDP(1, whole, fraction)
 #define FIXED_2DP(whole, fraction) FIXED_XDP(10, whole, fraction)
 
-// Construct a money value in the format MONEY(10,70) to represent 10.70. Fractional part must be two digits.
-#define MONEY(whole, fraction) ((whole)*10 + ((fraction) / 10))
+// Convert money literal to money32
+constexpr money32 operator"" _GBP(long double money)
+{
+    return money * 10;
+}
+// Helper function to convert runtime values to money
+constexpr auto MONEY(int32_t whole, int32_t fraction)
+{
+    return (whole)*10 + ((fraction) / 10);
+}
 
-#define MONEY_FREE MONEY(0, 00)
+#define MONEY_FREE 0.00_GBP
```

Comment:
Please remove this define entirely - it does not add anything of value (pun not intended).